src/analysis/inner_product_space/basic.lean

```diff
@@ -1419,6 +1419,39 @@ begin
     ... ≤ ∥innerSL x∥ * ∥x∥ : (innerSL x).le_op_norm _ }
 end
 
+/-- The inner product as a continuous sesquilinear map, with the two arguments flipped. -/
+def innerSL_flip : E →L[𝕜] E →L⋆[𝕜] 𝕜 :=
```

Comment:
`SL` looks a lot like "special linear" in my eyes. What do you think of `innersesq_flip`?
(I understand it's mostly an implementation detail...)

Reply:
That was inspired by the `E →SL[σ] F` notation for semilinear maps, but I see your point. @hrmacbeth what do you think?

Reply:
My eyes are also very used to the `E →SL[σ] F` notation, so I didn't have the associations that @jcommelin did. I rather like it because it's parallel with the `innerₛₗ` naming for the other one.  Do you mind, @jcommelin?

If we change, note that we have used the token `innerSL` in several declarations previously, so it should be a change throughout the file rather than just for this one declaration.

Reply:
Well, in that case it's fine. And even if we want to change it, certainly that can wait for another PR.